.scala-steward.conf

```diff
@@ -22,10 +22,9 @@
 #     while the time parts are only used to abide to the frequency of
 #     the given expression.
 #
-# Default: @asap
 #
-#pullRequests.frequency = "0 0 ? * 3" # every thursday on midnight
-#pullRequests.frequency = "7 days"
+pullRequests.frequency = "0 0 ? * 0" # every monday at midnight
```

Comment:
this might be my fault, but the comment/reference code there seems to be wrong and has the wrong int representing thursday (which should be `4` instead of `3`). So we would want `1` here to represent Monday. (Seems like you can also use `MON`, for clarity) I double checked these with this site: https://crontab.guru/#0_0_*_*_MON

Reply:
Oof, good catch. Updated.